python/core/auto_generated/qgsimagecache.sip.in

```diff
@@ -35,7 +35,7 @@ QgsImageCache is not usually directly created, but rather accessed through
 Constructor for QgsImageCache, with the specified ``parent`` object.
 %End
 
-    QImage pathAsImage( const QString &path, const QSize size, const bool keepAspectRatio, const double opacity, bool &fitsInCache /Out/ );
+    QImage pathAsImage( const QString &path, const QSize size, const bool keepAspectRatio, const double opacity, bool &fitsInCache /Out/, bool synchrone = false );
```

Comment:
"synchrone" -> "blocking" (English language, matches api usage elsewhere in QGIS). Also don't forget to document what the new argument does.

Reply:
Do you mean, you prefer "blocking" as "synchrone" ?

Reply:
done